Sources/TracingInstrumentation/TracingInstrument.swift

```diff
@@ -25,7 +25,7 @@ public protocol TracingInstrument: Instrument {
     ///   - timestamp: The `DispatchTime` at which to start the new `Span`. Passing `nil` leaves the timestamp capture to the underlying tracing instrument.
     func startSpan(
         named operationName: String,
-        context: BaggageContext,
+        context: BaggageContextProtocol,
```

Comment:
👍 startSpan _is_ end user visible API, so here the protocol makes sense IMO.

Reply:
Updated it to accept a `BaggageContextCarrier` instead.